spark/src/test/java/org/apache/iceberg/spark/data/parquet/vectorized/TestParquetVectorizedReads.java

```diff
@@ -48,11 +48,15 @@
 import static org.apache.iceberg.types.Types.NestedField.required;
 
 public class TestParquetVectorizedReads extends AvroDataTest {
-  private static final int NUM_ROWS = 1_000_000;
+  private static final int NUM_ROWS = 200_000;
 
   @Override
   protected void writeAndValidate(Schema schema) throws IOException {
-    writeAndValidate(schema, NUM_ROWS, 0L, RandomData.DEFAULT_NULL_PERCENTAGE, false, true);
+    writeAndValidate(schema, getNumRows(), 0L, RandomData.DEFAULT_NULL_PERCENTAGE, false, true);
+  }
+
+  protected int getNumRows() {
+    return NUM_ROWS;
```

Comment:
Why make this a method? So it can be overridden?

Reply:
Yes